components/KitBackgroundScreen.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { View, StyleSheet, Image, TouchableOpacity } from "react-native";
+import Colors from "../constants/Colors";
+import KitText from "./KitText";
+
+export default function KitBackgroundScreen(props) {
+  return (
+    <View style={styles.background}>
+      <View style={styles.main}>
+        <View>
+          <KitText style={styles.title} size={26}>
+            {props.title}
+          </KitText>
+          <TouchableOpacity
+            style={styles.backButton}
+            onPress={props.onPressBack}
+          >
+            <Image source={require("../assets/images/arrow.png")} />
+          </TouchableOpacity>
+        </View>
+        <View style={styles.container}>{props.children}</View>
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  background: {
+    flex: 1,
+    backgroundColor: Colors.KIT_GREEN,
```

Comment:
I imagine the same `KitBackgroundScreen` component can be used for other screen such as the `SpecficTeamsScreen`. 
![image](https://user-images.githubusercontent.com/23728476/69589398-d3c04a80-0fa0-11ea-96e4-be359a673f60.png).

Can we change backgroundColor to be a passed prop?

Reply:
good catch, will do